src/palace/manager/sqlalchemy/model/coverage.py

```diff
@@ -109,10 +111,8 @@ class Timestamp(Base):
     COVERAGE_PROVIDER_TYPE = "coverage_provider"
     SCRIPT_TYPE = "script"
 
-    # A stand-in value used to indicate that a field in the timestamps
-    # table should be explicitly set to None. Passing in None for most
-    # fields will use default values.
-    CLEAR_VALUE = object()
+    # We use SentinelType.ClearValue as a stand-in value used to indicate that a field in the timestamps
+    # table should be explicitly set to None. Passing in None for most fields will use default values.
```

Comment:
Update this to use our `SentinelType` enum, since we can type check it that way.